### ISSUE TITLE
Unsaved LegalUser objects have incorrect person_type

### ISSUE THREAD

**Issue by Changaco** [Mangopay/mangopay2-python-sdk#118]
```python
>>> from mangopay.resources import LegalUser
>>> LegalUser().person_type
'NATURAL'
```

The result should be `'LEGAL'` instead of `'NATURAL'`.

**Comment by mickaelpois** [comment]
Hello @Changaco,

Thanks for your feedback, we will fix it very soon, as it is a tiny but important mistake :)

Mickaël

**Comment by mickaelpois** [comment]
Fixed by #128
